### PR TITLE
Mwpw-142267:  Merch What's Included and Merch Mnemonic List (TwP)

### PR DESCRIPTION
**Pull request by Axelcureno**
Introduces Merch What's Included and Merch Mnemonic List web components for TwP What's included screen.

Resolves: [MWPW-140338](https://jira.corp.adobe.com/browse/MWPW-140338)

Includes: https://github.com/adobecom/mas/pull/4/files

**Test URLs:**
- Before: https://main--milo--adobecom.hlx.page/drafts/axel/block-samples/twp/illustrator#twpmodal?martech=off
- After: https://mwpw-142267--milo--adobecom.hlx.page/drafts/axel/block-samples/twp/illustrator#twpmodal?martech=off

### CODE REVIEW

File: libs/deps/merch-mnemonic-list.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
+// branch: MWPW-142267 commit: db56fa6d1f49aa580f3da94842ffb9e73516ee18 Mon, 08 Jul 2024 21:06:50 GMT
```
Comment (3ch023) [inline libs/deps/merch-mnemonic-list.js:1]: since this PR was already merged in mas repository these files should now be regenerated from /main branch

**Review by 3ch023 — APPROVED** [review]

File: libs/deps/merch-mnemonic-list.js (hunk 1 above)
Comment (Axelcureno) [reply libs/deps/merch-mnemonic-list.js:1]: Not sure why it still says branch is different than main but these were generated from main

File: libs/blocks/merch-twp/merch-twp.js [hunk 2]
```diff
@@ -72,6 +72,26 @@ export default async function init(el) {
...
+  if (whatsIncludedFragment) {
+    const whatsIncludedSlot = createTag(
+      'div',
+      {
+        slot: 'merch-whats-included',
+        class: 'hidden merch-whats-included-container',
+      },
+      whatsIncludedFragment,
+    );
+    twp.append(whatsIncludedSlot);
```
Comment (npeltier) [inline libs/blocks/merch-twp/merch-twp.js:85]:
```suggestion
      whatsIncludedFragment,
      { parent : twp },
    );
```

File: libs/deps/merch-card.js [hunk 3]
```diff
@@ -1,7 +1,6 @@
-// branch: main commit: 258ad3d851cc4945eae2aab2dcc80a8d0c14d861 Fri, 28 Jun 2024 14:19:21 GMT
-import{html as a,LitElement as j}from"/libs/deps/lit-all.min.js";import{LitElement as q,html as T,css as F}from"/libs/deps/lit-all.min.js";var s=class extends q{static properties={size:{type:String,attribute:!0},src:{type:String,attribute:!0},alt:{type:String,attribute:!0},href:{type:String,attribute:!0}};constructor(){super(),this.size="m",this.alt=""}render(){let{href:e}=this;return e?T`<a href="${e}">
+import{html as o,LitElement as te,nothing as re}from"/libs/deps/lit-all.min.js";import{LitElement as Y,html as M,css as Q}from"/libs/deps/lit-all.min.js";var h=class extends Y{static properties={size:{type:String,attribute:!0},src:{type:String,attribute:!0},alt:{type:String,attribute:!0},href:{type:String,attribute:!0}};constructor(){super(),this.size="m",this.alt=""}render(){let{href:e}=this;return e?M`<a href="${e}">
```
Comment (npeltier) [inline libs/deps/merch-card.js:1]: please use milo build cc @3ch023

Comment (npeltier) [reply libs/deps/merch-card.js:1]: (we need to see branch & commit hash)

File: libs/deps/merch-mnemonic-list.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
+import{html as e,css as i,LitElement as s}from"/libs/deps/lit-all.min.js";var t=class extends s{static styles=i`
```
Comment (npeltier) [inline libs/deps/merch-mnemonic-list.js:1]: same

File: libs/deps/merch-twp-d2p.js [hunk 5]
```diff
@@ -1,4 +1,3 @@
-// branch: develop commit: b3f6608faa10db8d0187b310044d4690d063f1bf Sun, 09 Jun 2024 06:39:58 GMT
```
Comment (npeltier) [inline libs/deps/merch-twp-d2p.js:1]: same

File: libs/deps/merch-whats-included.js [hunk 6]
```diff
@@ -0,0 +1,44 @@
+import{html as e,css as o,LitElement as l}from"/libs/deps/lit-all.min.js";var t=class extends l{static styles=o`
```
Comment (npeltier) [inline libs/deps/merch-whats-included.js:1]: same

**Review by npeltier — CHANGES_REQUESTED** [review]

File: libs/deps/merch-whats-included.js (hunk 6 above)
Comment (3ch023) [reply libs/deps/merch-whats-included.js:1]: addressed

File: libs/blocks/merch-twp/merch-twp.js (hunk 2 above)
Comment (3ch023) [reply libs/blocks/merch-twp/merch-twp.js:85]: addressed

**Review by npeltier — APPROVED** [review]

**Comment by Axelcureno** [comment]
@Roycethan please validate this one 🙏🏼

**Comment by Roycethan** [comment]
> @Roycethan please validate this one 🙏🏼

@Axelcureno  In the implemented modal the below AC's are not implemented, hence assigning back to you thanks
AC: Ability for authors to add CTA which will take users to the cart
AC: Ability for that CTA to know what the user's subscription option was from the previous screen (nice to have)
AC: Back CTA so that users can go back to the previous screen (Note: back cta needs to be smart in knowing if the user was on screen 1 or a 2nd screen in the case of modal with 2 screens already)

**Comment by Roycethan** [comment]
The remaining un-covered AC's and Issues are noted in https://jira.corp.adobe.com/browse/MWPW-154370 and moving forward with covered use cases for facilitating authoring...

**Pull request merged**